include/bx/inline/bx.inl

```diff
@@ -7,6 +7,10 @@
 #	error "Must be included from bx/bx.h!"
 #endif // BX_H_HEADER_GUARD
 
+#if __cplusplus >= BX_LANGUAGE_CPP20 && !BX_CRT_NONE
+#	include <bit>
```

Comment:
Just in case it's not obvious. bx doesn't include any of STD headers.

Reply:
It wasn't immediately apparent, but upon a quick [check](https://beyondgrep.com/) of the code base, I noticed the use of `<type_traits>` [here](https://github.com/bkaradzic/bx/blob/67dfdf34f642a4a807b75eb600f82f4f04027963/src/dtoa.cpp#L11). This served as a precedent for me to suggest this approach. No harm done. Similarly, the `BX_LANGUAGE_CPP20` macro was used in response to its existence.
It might be helpful to establish `CONTRIBUTING` guidelines to avoid confusion in the future.